docker-compose.yml

```diff
@@ -4,16 +4,20 @@ services:
   files:
     image: files
     volumes:
-      - /data/dumps/path:/input # change /data/dumps/path to the location of the RDF data dumps, make sure to add the dumps to files/config.json as well
-      - /fragment/path:/output  # change /fragment/path to the desired location of the fragments
+      - /home/hdelva/termen:/input # change /data/dumps/path to the location of the RDF data dumps, make sure to add the dumps to files/config.json as well
```

Comment:
Perhaps change this to `/data/dumps/path`, to become independent of Harm? :)

Reply:
Changed now, thanks! Forgot to revert some changes after testing the compose file.